lib/irc/ClientPool.js

```diff
@@ -284,7 +284,7 @@ ClientPool.prototype._onClientDisconnected = function(bridgedClient) {
     // the timeout below holds it in a closure, preventing it from being GC'd.
     bridgedClient = undefined;
 
-    if(chanList.length === 0) {
+    if (chanList.length === 0) {
```

Comment:
Boo, unrelated to the commit.

Reply:
I know :disappointed: . It was just a space :s

Reply:
No probs. `git add -p` is your friend.